packages/pigeon/lib/dart_generator.dart

```diff
@@ -135,6 +135,8 @@ void generateDart(Root root, StringSink sink) {
       '// ignore_for_file: public_member_api_docs, non_constant_identifier_names, avoid_as, unused_import');
   indent.writeln('import \'dart:async\';');
   indent.writeln('import \'package:flutter/services.dart\';');
+  indent.writeln(
```

Comment:
Unlike the top 2, this is conditional right? Otherwise we might get unused import lint if the schema doesn't call for binary data anywhere?

Reply:
A couple of lines higher I disable the unused_import lint.

Reply:
Rather than disabling the linter, can we just conditionally add this line if binary data is used in the schema?

Reply:
We could and I thought about it previously.  In this case I think keeping the code more simple outweighs the suppression of an unused import warning.